modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/viewmodel/PodcastViewModel.kt

```diff
@@ -380,6 +380,19 @@ class PodcastViewModel
         settings.setBookmarksSortType(order)
     }
 
+    fun play(bookmark: Bookmark) {
+        val time = bookmark.timeSecs
+        val bookmarkEpisode = (uiState.value as? UiState.Loaded)?.episodes?.firstOrNull { it.uuid == bookmark.episodeUuid }
+        bookmarkEpisode?.let {
+            val shouldPlayEpisode = !playbackManager.isPlaying() ||
+                playbackManager.getCurrentEpisode()?.uuid != bookmarkEpisode.uuid
+            if (shouldPlayEpisode) {
+                playbackManager.playNow(it, sourceView = SourceView.PODCAST_LIST)
+            }
+        }
+        playbackManager.seekToTimeMs(time * 1000)
```

Comment:
Nitpick: I would inline this `time` variable since it's not used anywhere else and `seekTimeToMs(bookmark.timeSecs * 1000)` seems nice since it makes the seconds to ms conversion more explicitly. Certainly not a big deal though.

Reply:
Nice pick. 🙂

Updated in https://github.com/Automattic/pocket-casts-android/pull/1254/commits/e3031f750d3309994ba2e18ea60a4398778c7ef2